source/index.html.md

```diff
@@ -820,6 +874,66 @@ To determine if a line item was matched, use the productCodeMatched attribute:
+| files.packingList.weightGrossTotal      | The packing lists total gross weight. |
+| files.packingList.weightNetTotal        | The packing lists total net weight. |
+| files.packingList.volumeTotal           | The packing lists total volume. |
+| files.packingList.weightUnit            | The packing lists weight unit. |
+| files.packingList.volumeUnit            | The packing lists volume unit. |
+| files.packingList.packageUnit           | The packing lists package unit type. |
+| files.packingList.itemUnit              | The packing lists item unit. |
+| files.packingList.itemQtyTotal          | The packing lists total item quanity. |
+| files.packingList.id                    | The internal ID of the packing list. |
+
+### *Files/packingList/lineItem* attributes
+
+#### Using the lineItem attributes to determine the product code and description
+The attributes extracted from an invoice for each line item (eg. Product code, description, HS Code, Origin and Unit) are available in the attributes `productCode`, `description`, `hsCode`, `originCountryCode`, `unitType`.
+
+When a line item is successfully matched to a product code from your company’s product code database, the matched values, taken from your product code database, are available in the ‘matched’ attributes (eg. `matchedproductCode`, `matchedDescription`, `matchedHsCode`, `matchedoriginCountryCode`, `matchedUnitType`)
+
+To determine if a line item was matched, use the productCodeMatched attribute:
+
+`productCodeMatched=true` - The line item was successfully matched
+`productCodeMatched=false` - the line item was not matched
+
+| Attribute                            | Description                               |
+| --------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------  |
+| files.packingList.lineItem.packingListId   | The internal ID of the line items packing list. |
+| files.packingList.lineItem.description     | The description of the line item. |
+| files.packingList.lineItem.itemQty         | The line items total item quantity. |
+| files.packingList.lineItem.packageQty      | The line items total package quantity. |
+| files.packingList.lineItem.netWeight       | The line items net weight. |
+| files.packingList.lineItem.grossWeight     | The line items gross weight. |
+| files.packingList.lineItem.volume          | The line items volume. |
+| files.packingList.lineItem.productCode     | The code for the product in the line item. |
+| files.packingList.lineItem.hsCode          | The HS Code of this line item. |
+| files.packingList.lineItem.id              | The internal ID of the line item. |
+| files.packingList.lineItem.orderIndex      | The index of the line, representing the order of it within the commercial invoice. |
+| files.packingList.lineItem.descriptionCell | The entire cell of the line item description. |
```

Comment:
Remove this one please, in the changes I've made for this api in my pr this one is no longer present